### PR TITLE
Feature - Activity stream for Partners, Intervention, and Government Intervention

### PR DESCRIPTION
**Pull request by n1207n**
This PR addresses a refactoring for create_snapshot_activity_stream method as well as adding activity stream support for Partners, Intervention, and Government Intervention models.

### CODE REVIEW

File: EquiTrack/EquiTrack/stream_feed/actions.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+            if not delta_dict:
+                # Restore the previous state of current object by merging above
+                previous = dict(current_obj_dict.items() + changed_prev_values.items())
+
+                # Extract current field changes from key lookups with current object
+                changes = {k:v for k,v in current_obj_dict.items() if k in changed_prev_values}
+
+            else:
+                previous = current_obj_dict
+                changes = delta_dict
```
Comment (robertavram) [inline EquiTrack/EquiTrack/stream_feed/actions.py]: delta dict should only contain fields that are properties of the model.. and also only those fields that are actually changed.

**Comment by n1207n** [comment]
@ntrncic, just finished resolving the conflicts.

**Comment by robertavram** [comment]
@n1207n sorry for delays on this merge.. could you update from develop one more time
Also, reach out to me on slack when you get in

File: EquiTrack/partners/views/v2.py [hunk 2]
```diff
@@ -3,6 +3,7 @@
 
 from django.shortcuts import get_object_or_404
 from django.contrib.auth import models
+
```
Comment (robertavram) [inline EquiTrack/partners/views/v2.py]: keep this file as is on the server unless you need it

Comment (n1207n) [inline EquiTrack/partners/views/v2.py]: Fair point -- But at some point we really need to clean up the code formatting at all scale.

**Pull request merged**